README.md

```diff
@@ -41,22 +44,36 @@ valid Dotty code from compiling.
 A tool to port code from Scala 2.x to Dotty is currently in development at
 https://github.com/scalacenter/scalafix
 
-build.properties
-----------------
-```
-sbt.version=0.13.13
+If your build contains dependencies that have only been published for Scala 2.x,
+you may be able to get them to work on Dotty by replacing:
+
+```scala
+    libraryDependencies += "a" %% "b" % "c"
 ```
 
-plugins.sbt
------------
+by:
+
+```scala
+    libraryDependencies += ("a" %% "b" % "c").withDottyCompat()
 ```
-addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.1.0-RC3")
+
+This will have no effect when compiling with Scala 2.x, but when compiling
+with Dotty this will change the cross-version to a Scala 2.x one. This
+works because Dotty is currently retro-compatible with Scala 2.x.
```

Comment:
No, we might switch to 2.12 at some point, e.g. dotty 0.1 with 2.11.x and dotty 0.2 with 2.12.x